This works around https://bugs.debian.org/967954 while bindgen 0.55 is
still the only version in unstable.

When bindgen gets updated in unstable, we can continue to relax this.

(it'd be even better for bindgen to release a 1.x.x version!)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,7 +28,7 @@
 
 [dependencies]
 [build-dependencies.bindgen]
-version = ">= 0.53.1, < 0.58.0"
+version = ">= 0.53.1, < 0.56.0"
 features = ["runtime"]
 default-features = false
 
